Tests/UITests/AlertPresentableSpec.swift

```diff
@@ -0,0 +1,85 @@
+import XCTest
+import Quick
+import Nimble
+import class Shock.MockServer
+
+class AlertPresentableSpec: QuickSpec {
+
+    override func spec() {
+
+        var mockServer: MockServer!
+        var app: XCUIApplication!
+
+        func launchAppIfNecessary(context: String) {
+            mockServer.setup(route: MockServerHelper.pingRouteMock(jsonStub: context))
+            mockServer.start()
+
+            guard app == nil else {
+                return
+            }
+            app = XCUIApplication()
+            app.launchArguments.append("--uitesting")
+            app.launchArguments.append("-context \(context)")
+            app.launch()
+        }
+
+        beforeEach {
+            mockServer = MockServerHelper.setupNewServer(route: MockServerHelper.standardRouting)
+        }
+
+        afterEach {
+            mockServer.stop()
+            app = nil
```

Comment:
In other UITest specs the condition in line 17 looks like this:
`guard app == nil || !app.launchArguments.joined(separator: " ").contains(context) else {`
Thanks to that, there's no need to restart the app with `app = nil` (which takes time).
Did you try to use this approach here?

Reply:
Yes, I should use this approach. Updated here [f2b229d](https://github.com/rakutentech/ios-inappmessaging/pull/269/commits/f2b229de48740aada9365a28a8f1438f8ef8ca8d).